.gitignore

```diff
@@ -94,3 +94,8 @@ __pycache__
 
 # Application data
 /var
+
+**/target
+**/.next
+
+seed-users.json
```

Comment:
Let’s copy these new lines to `.prettierignore` and `.markdownlintignore`. There's a section called

```
########################
## Same as in .gitignore
########################
```

Reply:
Thanks, addressed in a85c42aabdda2b11903f2f69a1293201cf506587
also moved the entries to the right place :)

Reply:
Rando question: do we really need to do things like specify `.json` files in `.markdownlintignore` @kachkaev? Surely the MD linter is never running on JSON files?